cfme/automate/import_export.py

```diff
@@ -36,6 +36,8 @@ def is_displayed(self):
 
 
 class AutomateImportExportView(AutomateImportExportBaseView):
+    flash = Text("//div[contains(@class, 'import-flash-message')]")
```

Comment:
Q: can this use the `FlashMessages` widget? May have to subclass to overwrite locators/xpath.

Reply:
Hey @john-dupuy 
- I agree on using ```FlashMessage``` or ```FlashMessages``` to assert the error message. I tried these widgets with locator ```//div[contains(@class, 'import-flash-message')]``` but it does not read the message.
- So I have changed the name from flash to error_flash for this widget to avoid override of flash widget from BaseClass.
- Please let me know your view.

Reply:
@ganeshhubale I would have to check to make sure, but I think you can subclass and modify the `messages` xpath locator @sshveta did something similar with `V2VFlashMessages`

https://github.com/ManageIQ/integration_tests/blob/8ae591f505e95cb0291258fc47987c4ec33efb66/widgetastic_manageiq/__init__.py#L5864-L5876